src/jpeg.imageio/jpeg_pvt.h

```diff
@@ -32,6 +32,8 @@ extern "C" {
 #    define OIIO_JPEG_LIB_VERSION JPEG_LIB_VERSION
 #endif
 
+#include "ultrahdr_api.h"
```

Comment:
Certainly this should only happen if the ultrahdr library was found at build time.

Reply:
Completely agree, I made the `libuhdr` dependency optional and updated the code accordingly :)